src/ua/griddynamics/Game.java

```diff
@@ -0,0 +1,207 @@
+package ua.griddynamics;
+
+import java.util.Random;
+
+public class Game {
+    static final int WIDTH = 9;
+    static final int HEIGHT = 9;
+
+    public final char X = 'X';
+    public final char FREE = '/';
+    public final char MINE = '*';
+    public final char POINT = '.';
+
+    private final int numberOfMines;
+    private final Cell[][] fields;
+
+    public Game(int numberOfMines) {
+        this.fields = new Cell[WIDTH][HEIGHT];
+        for (int i = 0; i < fields.length; i++) {
+            for (int j = 0; j < fields[i].length; j++) {
+                fields[i][j] = new Cell();
+            }
+        }
+        this.numberOfMines = numberOfMines;
+    }
+
+    public void placeMines(Coordinate coordinate) {
+        Random random = new Random();
+        int minesPlaced = 0;
+        int userX = coordinate.getX();
+        int userY = coordinate.getY();
+        do {
+            int x = random.nextInt(WIDTH);
+            int y = random.nextInt(HEIGHT);
+            if (!fields[y][x].isMine() && (x != userX || y != userY)) {
+                fields[y][x].mine = true;
+                minesPlaced++;
+            }
+        } while (minesPlaced < numberOfMines);
+        numberGenerator();
+    }
+
+    public String getField() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(" |123456789| ");
+        sb.append(System.lineSeparator());
+        sb.append("-|---------|");
+        sb.append(System.lineSeparator());
+        int rowCounter = 1;
+        for (Cell[] field : fields) {
+            sb.append(rowCounter).append("|");
+
+            for (int j = 0; j < fields.length; j++) {
+                sb.append(getSymbol(field[j]));
+            }
+            sb.append("|");
+            sb.append(System.lineSeparator());
+            rowCounter++;
+        }
+        sb.append("-|---------|");
+        return sb.toString();
+    }
+
+    private char getSymbol(Cell cell) {
+        if (cell.isChecked()) {
+            if (cell.markedAsMine) {
+                return MINE;
+            }
+            if (cell.markAsFree) {
+                return FREE;
+            }
+            if (cell.isMine()) {
+                return X;
+            }
+            if (!cell.isEmpty() && !cell.isMine()) {
+                return (char) (cell.getNeighborMines() + 48);
```

Comment:
Why 48?
use some specific char for this, e.g.` (a+'0')`

Reply:
Done